Assets/MixedRealityToolkit/Definitions/InputSystem/MixedRealityRaycastHit.cs

```diff
@@ -1,6 +1,10 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+// Including lightmap coord has a performance cost so we've disabled it by default.
+// Uncomment the define to enable lightmap coords.
+// #define MRTK_ENABLE_LIGHTMAPCOORD_IN_RAYCASTHIT
```

Comment:
I know that I let the other change languish - I'm going to get that fixed today. Can you leave this file untouched so that my change can just do its part?

Reply:
Does your change avoid a Unity.Object reference null check?

Reply:
I'm going to change it to be a struct, so no more null checks.